home.php

```diff
@@ -3,5 +3,23 @@
 echo "This is home page";
 // assign value
 $title = "Welcome to this website";
+# get menu
+$menu = "home_menu";
+
+function getMenuIndex($name){
+	if ($name = "home_menu"){
+		return 1;
+	}
+	return 2;
+}
+
+function getTotal($a, $b){
```

Comment:
__[phpmd]__
* Avoid variables with short names like $a. Configured minimum length is 3.
* Avoid variables with short names like $b. Configured minimum length is 3.

<!-- Comment by [SideCI](https://sideci.com) -->